manifests/init.pp

```diff
@@ -98,10 +92,8 @@
   String $group = 'pulp',
   Stdlib::Absolutepath $user_home = '/var/lib/pulp',
   Boolean $manage_apache = true,
-  Stdlib::Host $api_host = '127.0.0.1',
-  Stdlib::Port $api_port = 24817,
-  Stdlib::Host $content_host = '127.0.0.1',
-  Stdlib::Port $content_port = 24816,
+  Pattern['^unix://'] $api_service_bind = 'unix:///run/pulpcore-api.sock',
```

Comment:
The Pulp project itself is using the path `/var/run/pulpcore-api` (https://github.com/pulp/pulp_installer/blob/d6fd0f468f7773d01d7eb146044daec5b9c74a18/molecule/source-static/group_vars/all#L4). Given this socket is shared between the pulpcore systemd socket and Apache, and is owned by the Apache user I leaned towards a top level location. However, I am open to congruency with the Pulp project.

Reply:
I personally stopped using `/var/run`. It's deprecated in favor of `/run` and just a symlink nowadays. However, in practice it means there is no real difference.

Reply:
Sorry, I should have been more clear what I was asking here. The variation comes in the directory under which the socket is stored, Pulp uses the full path `/run/pulpcore-api/pulpcore-api.sock` (note the scoping on the `pulpcore-api` directory).

Reply:
Ah, that difference. Given that it's owned by Apache and Apache may not have permission to read in `/run/pulpcore-api`, I'm also leaning to a top level location. Inside the Apache run dir may also not be correct since I'm not sure you can rely on that existing when the socket is started.